homeassistant/components/zha/core/channels/base.py

```diff
@@ -97,6 +95,12 @@ def __init__(
         unique_id = ch_pool.unique_id.replace("-", ":")
         self._unique_id = f"{unique_id}:0x{cluster.cluster_id:04x}"
         self._report_config = self.REPORT_CONFIG
+        if not hasattr(self, "_value_attribute") and len(self._report_config) > 0:
```

Comment:
I propose the very 1st attribute configured for reporting to be considered the "value" attribute. But  in the long run would like to phase out the "value" attribute, as the lines are blurred when a channel should have multiple entities, eg "energy metering" channel

Reply:
Completely agree. That’s what I’m doing here. Thought was let’s keep it working the way it does now here. Get the new signature in place (attrid, name, value) then in the next pr handle multiple reads at the same time and remove value attribute.